src/AudioInterface.h

```diff
@@ -98,15 +98,17 @@ class AudioInterface
      * \param NumInChans Number of Input Channels
      * \param NumOutChans Number of Output Channels
      * \param AudioBitResolution Audio Sample Resolutions in bits
+     * \param processWithNetwork Send audio to and from the network
+     * \param jacktrip Pointer to the JackTrip class that connects all classes (mediator)
```

Comment:
The jacktrip parameter is redundant here. Why did you change the order of parameters?

Reply:
I probably didn't need to. It was really only in RtAudioInterface and JackAudioInterface that it made an actual difference, but it does seem to make sense for consistency and clarity that the constructor parameter order for the base class should match that used in the derived classes.

And the change was to get rid of an overloaded constructor in the derived classes that was needlessly complicating things. It just ran the original constructor but with a nullptr as the jacktrip pointer. (Because it's being run directly from vsAudioInterface.) This constructor also set processWithNetwork to false without any documentation of this. Which means we had two constructors which look like they do the same thing (just with effectively a default nullptr being passed to the parent class) but which actually have different default behaviours. That's just asking for things to break further down the line.

Reply:
Yeah, that sounds bad. To me it felt right to have the jacktrip pointer at pole position, but I don't care. Could you remove the redundant \param? Didn't test it myself, but sounds like it's well enough tested that we can merge this.

Reply:
Yeah, I kind of agree on that. But it's not uncommon with the Qt classes for the parent pointer to be one of the final arguments, so that at least fits with that kind of paradigm. Was wondering what you were on about with the redundant param and then realised I left the first one in there. Will fix that now. (Thanks for spotting that.)

Reply:
Good to know. Seeing `this` as the first (or actually second?) argument in this context is confusing too. So maybe it's not that bad of an idea to put it in the end.